test/Microsoft.ML.Tests/Scenarios/Api/CookbookSamples/CookbookSamplesDynamicApi.cs

```diff
@@ -506,6 +506,45 @@ public class CustomMappings : CustomMappingFactory<InputRow, OutputRow>
             }
         }
 
+        public class IrisData
+        {
+            public float Label { get; set; }
+            public float SepalLength { get; set; }
+            public float SepalWidth { get; set; }
+            public float PetalLength { get; set; }
+            public float PetalWidth { get; set; }
+        }
+
+        [Fact]
+        public void ReadAndWriteBinaryData() =>
+            BinaryData();
+
+        private void BinaryData()
```

Comment:
You can collapse this together and just have the one function because it's not parameterized.